scripts/update_plugin_modules.sh

```diff
@@ -32,7 +32,7 @@ do
 		cd ..
 	fi
 	cd vault
-	go get github.com/hashicorp/$plugin@master
+	go get github.com/hashicorp/$plugin@main
```

Comment:
I wonder if we should do `... || go get github.com/hashicorp/$plugin@master` as a fallback mechanism in the near term in case this updated script gets run before all repos are updated.

Reply:
Do we even use this script? If so, I'd rather see the failure so we can go fix them (the rename is a button click in the Github UI).